cmd/worker/main.go

```diff
@@ -133,11 +155,15 @@ func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blo
 		}
 	}
 
+	if err = publishNotification(ctx, topicNotification, pkgIdentifier{name, version, ecosystem}); err != nil {
```

Comment:
We can have `publishNotification` take in the raw name, version and ecosystem strings. and  have it construct the pkgIdentifier before marshalling it to JSON.